forge-gui/res/cardsfolder/upcoming/deadly_cover_up.txt

```diff
@@ -0,0 +1,15 @@
+Name:Deadly Cover-Up
+ManaCost:3 B B
+Types:Sorcery
+S:Mode$ OptionalCost | EffectZone$ All | ValidCard$ Card.Self | ValidSA$ Spell | Cost$ CollectEvidence<6> | Description$ As an additional cost to cast this spell, you may collect evidence 6. (Exile cards with total mana value 6 or greater from your graveyard.)
+A:SP$ DestroyAll | ValidCards$ Creature | SubAbility$ DBExile | SpellDescription$ Destroy all creatures.
+SVar:DBExile:DB$ ChangeZone | ChangeType$ Card.OppCtrl | Hidden$ True | Origin$ Graveyard | Destination$ Exile | RememberChanged$ True | ConditionDefined$ Collected | ConditionPresent$ Card | SubAbility$ ExileYard | SpellDescription$ If evidence was collected, exile a card from an opponent's graveyard. Then search its owner's graveyard, hand, and library for any number of cards with that name and exile them. 
+SVar:ExileYard:DB$ ChangeZone | ChangeType$ Remembered.sameName | Origin$ Graveyard | DefinedPlayer$ RememberedController | Chooser$ You | ConditionDefined$ Collected | ConditionPresent$ Card | Destination$ Exile | ChangeNum$ NumInYard | Hidden$ True | SubAbility$ ExileHand 
+SVar:ExileHand:DB$ ChangeZone | Origin$ Hand | Destination$ Exile | DefinedPlayer$ RememberedController | ChangeType$ Remembered.sameName | ConditionDefined$ Collected | ConditionPresent$ Card | ChangeNum$ NumInHand | Chooser$ You | SubAbility$ ExileLib | StackDescription$ None
```

Comment:
Don't think `RememberedController `is correct here ? It doesn't seem to work after the chosing of card in opponents grave

Reply:
you messed up the ChangeNum SVar